CharlieBackend.Api/appsettings.json

```
@@ -7,11 +7,11 @@
     }
   },
   "AllowedHosts": "*",
-    "ConnectionStrings": {
-      "DefaultConnection": null,
-      "RabbitMQ": null,
-      "AzureBlobsAccessKey": null
-    },
+  "ConnectionStrings": {
+    "DefaultConnection":null,
```

```suggestion
    "DefaultConnection": null,
```
If you're removing redundant spaces then keep this one